test/blackbox/annotations.bats

```diff
@@ -1,21 +1,44 @@
-load helpers_pushpull
+load helpers_zot
+
+function verify_prerequisites {
+    if [ ! -f ${ZOT_PATH} ]; then
+        echo "you need to build ${ZOT_PATH} before running the tests" >&3
+        return 1
+    fi
+
+    if [ ! $(command -v curl) ]; then
+        echo "you need to install curl as a prerequisite to running the tests" >&3
+        return 1
+    fi
+
+    if [ ! $(command -v jq) ]; then
+        echo "you need to install jq as a prerequisite to running the tests" >&3
+        return 1
+    fi
+
+    if [ ! $(command -v podman) ]; then
+        echo "you need to install podman as a prerequisite to running the tests" >&3
+        return 1
+    fi
+
```

Comment:
Don't we need to add checks or skopeo and stacker?

Reply:
This tools are already checked/installed if run through a Makefile target. If run as a standalone BATS script we need to make sure versions (ex: cosign.notation) are the same as the one installed by Makefile. This adds extra complexity that is probably not needed. We can add a note at the beginning of each file that states just that. Besides that there are some tests that did not pass locally anyway (that ones that requires crictl for ex or the sync one that syncs images from dockerhub - probably limitation of unpaid version)